packages/units/src/useUnits.js

```diff
@@ -17,7 +17,10 @@
 /**
  * Internal dependencies
  */
-import { identity, useContextSelector } from '../../design-system';
+import {
+  identity,
+  useContextSelector,
+} from '../../../assets/src/design-system';
```

Comment:
Dang, I forgot about this.

Relative imports from `assets/src` need to be avoided in packages.

The design system can soon be moved to a package in #6051, which currently is only blocked by #7824.

It should be OK to keep this here for now, but it should get fixed ASAP once `@web-stories-wp/design-system` is a thing.